CHANGELOG.md

```diff
@@ -941,6 +944,8 @@ This changelog documents all notable changes to VAST and is updated on every rel
 - The example configuration file contained an invalid section `vast`. This has been changed to the correct name `system`.
   [#705](https://github.com/tenzir/vast/pull/705)
 
+## [.DS_Store]
```

Comment:
I think that's not supposed to be here.

Reply:
I already force-pushed this out, thankfully the CI detected this as well. Stupid .DS_Store files.